### PR TITLE
feat: Pass debug trace options to dev tools

### LINKED ISSUE #678: Should configure redux dev tools to serialize map/set when map/set functionality is enabled.
Seeing as we are telling easy-peasy that we want to enable map/set functionality, it would make sense that redux dev tools should be configured to display them out of the box. I was surprised when most of my state wasn't appearing in dev tools.

A possible plan would be:

1. `devTools` store config can take a redux dev tools config object.
2. Pass that config when initializing redux dev tools https://github.com/ctrlplusb/easy-peasy/blob/5c4c0e8e0a708e0c218f82d0a89aa679d1f44dcb/src/create-store.js#L77
3. If no config object was set, and map/set is enabled, include something like this:
```js
  serialize: { 
    'map': true,
    'set': true,
  }
```

### PR DESCRIPTION
**Pull request by jmyrland**
continuation of #495 rebased on master

closes #678

### CODE REVIEW

**Review by w3bdesign — APPROVED** [review]

**Review by w3bdesign — DISMISSED** [review]

File: src/create-store.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ export function createStore(model, options = {}) {
   const modelClone = clone(model);
   const {
     compose,
-    devTools = process.env.NODE_ENV !== 'production',
```
Comment (jmyrland) [inline src/create-store.js:24]: Any type updates we need to consider here?
Should we support both bool and object input?

File: src/create-store.js [hunk 2]
```diff
@@ -76,6 +76,7 @@ export function createStore(model, options = {}) {
     window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__
       ? window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__({
           name: storeName,
+          ...devTools,
         })
```
Comment (jmyrland) [inline src/create-store.js:80]: Is this enough?

File: src/create-store.js (hunk 1 above)
Comment (ctrlplusb) [reply src/create-store.js:24]: Hmmm. Good point. Will have to review.

Comment (jmyrland) [reply src/create-store.js]: I think I've fixed it now - but I was unable to verify it 😕 

Tried to install `easy-peasy` by using `file:../../` as the version for the example - but I just got the `5.1.0` source. And yes, I did run `yarn build` before doing this.

File: examples/reduxtagram/src/store.ts [hunk 3]
```diff
@@ -4,7 +4,7 @@ import posts from '@/data/posts';
 import { storeModel } from '@/model';
 
 const store = createStore(storeModel, {
-  devTools: import.meta.env.NODE_ENV === 'development',
+  devTools: import.meta.env.DEV ? { name: 'This is a test' } : false,
```
Comment (ctrlplusb) [inline examples/reduxtagram/src/store.ts:7]: **question:** Is this ternary intentional?

**Comment by ctrlplusb** [comment]
Amazing, just a minor question. 👍

**Review by ctrlplusb — APPROVED** [review]
🚀

**Pull request merged**